script.js

```diff
@@ -9,6 +25,7 @@ var main = function (input) {
 };
 
 var scissorsPaperStone = function (userMove) {
+  totalGames++;
```

Comment:
good use of the ++ operator to increase the totalGames count by one every time the function runs!